sdk/daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Engine.scala

```diff
@@ -175,9 +178,9 @@ class Engine(val config: EngineConfig) {
           seeding = Engine.initialSeeding(submissionSeed, participantId, preparationTime),
           packageResolution = pkgResolution,
           engineLogger = engineLogger,
+          submissionInfo = Some(Engine.SubmissionInfo(participantId, submissionSeed, submitters)),
```

Comment:
For `submit`, we pass through extra submission information for use in saving `SubmissionEntry`